modules/nf-core/ragtag/patch/meta.yml

```diff
@@ -0,0 +1,110 @@
+name: "ragtag_patch"
+description: Patches a target assembly using a query assembly
```

Comment:
To be a 'naive' user: what does 'patching' mean here? Changing what for what? exactly

Reply:
From the tool's help:

Homology-based assembly patching: Make continuous joins and fill gaps in 'target.fa' using sequences from 'query.fa'